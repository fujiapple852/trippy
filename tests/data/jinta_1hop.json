[{
    "sequence": 1,
    "identifier": 1,
    "src_port": 80,
    "dest_port": 80,
    "ttl": 1,
    "round": 1,
    "sent": "2023-01-01T12:01:55.100",
    "status": "Complete",
    "host": "10.1.0.2",
    "received": "2023-01-01T12:01:55.300",
    "icmp_packet_type": null,
    "extensions": null
  },{
    "sequence": 2,
    "identifier": 1,
    "src_port": 80,
    "dest_port": 80,
    "ttl": 1,
    "round": 1,
    "sent": "2023-01-01T12:01:56.100",
    "status": "Complete",
    "host": "10.1.0.2",
    "received": "2023-01-01T12:01:56.200",
    "icmp_packet_type": null,
    "extensions": null
  },{
    "sequence": 3,
    "identifier": 1,
    "src_port": 80,
    "dest_port": 80,
    "ttl": 1,
    "round": 1,
    "sent": "2023-01-01T12:01:57.100",
    "status": "Complete",
    "host": "10.1.0.2",
    "received": "2023-01-01T12:01:57.250",
    "icmp_packet_type": null,
    "extensions": null
  },{
    "sequence": 4,
    "identifier": 1,
    "src_port": 80,
    "dest_port": 80,
    "ttl": 1,
    "round": 1,
    "sent": "2023-01-01T12:01:58.000",
    "status": "Complete",
    "host": "10.1.0.2",
    "received": "2023-01-01T12:01:59.001",
    "icmp_packet_type": null,
    "extensions": null
  }]