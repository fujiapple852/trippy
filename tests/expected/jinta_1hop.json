{
    "total_sent": 4,
    "total_recv": 4,
    "last": 1001,
    "best": 300,
    "worst": 700,
    "mean": 0.0,
    "m2": 0.0,
    "jitter": 851,
    "javg": 300.25,
    "jmax": 851,
    "jinta": 1148.74
}